src/components/cms/BrowseSkill/SideDrawer.js

```diff
@@ -63,8 +59,12 @@ export default function SideDrawer(props) {
 
   return (
     <div>
-      <Button onClick={toggleDrawer('left', true)} disable={false}>
-        <Icon className="material-icons">filter_list</Icon>
+      <Button
+        color="primary"
+        onClick={toggleDrawer('left', true)}
+        disable={false}
```

Comment:
Can this line be removed? as it has no purpose . Rest is fine

Reply:
Okay